autoload/neoformat.vim

```diff
@@ -92,7 +92,7 @@ function! s:neoformat(bang, user_input, start_line, end_line) abort
         endif
 
         " read from /tmp file if formatter replaces file on format
-        if cmd.replace
+        if cmd.replace || len(stdout) == 0
```

Comment:
It seems that this might result in some weird behavior when formatting buffers that are empty with a `stdin` based formatter.